src/components/home.vue

```diff
@@ -0,0 +1,17 @@
+<template>
```

Comment:
Why you did not use this component anywhere. It should be inside router with '/' path, similar to index component. This is first component that gets rendered when user enters the application.